e2e/test/iothub/DeviceClientX509AuthenticationE2ETests.cs

```diff
@@ -32,13 +32,6 @@ public DeviceClientX509AuthenticationE2ETests()
             _hostName = GetHostName(TestConfiguration.IoTHub.ConnectionString);
         }
 
-        [LoggedTestMethod]
-        [TestCategory("LongRunning")]
-        public async Task X509_InvalidDeviceId_Throw_UnauthorizedException_Amqp()
```

Comment:
Where we had tests for a protocol and for TcpOnly and WebSocketOnly, I removed the composite one. Otherwise, I changed them to use Tcp.